_docs/01-quick-start-guide.md

```diff
@@ -77,3 +77,55 @@ Send shutdown request to mqbroker(36695) OK
 The mqnamesrv(36664) is running...
 Send shutdown request to mqnamesrv(36664) OK
 ```
+
+
+# Windows 
+
+The guide is working for windows 10 , please make sure you have powershell installed.
+
+Download latest binary release. and extract zip file into your local disk. Such as: `D:\soft\rocketmq-all-4.7.0-bin-release`
```

Comment:
It is best not to be related to the specific version. IMO, `D:\rocketmq` would be better.